src/types.rs

```diff
@@ -883,3 +883,57 @@ where
     }
     deserializer.deserialize_any(HeightVisitor)
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Default)]
+pub struct WalletCreateFundedPsbtOptions {
+    #[serde(rename = "fee_rate", skip_serializing_if = "Option::is_none")]
+    pub fee_rate: Option<f64>,
+    #[serde(rename = "lock_unspent", skip_serializing_if = "Option::is_none")]
+    pub lock_unspent: Option<bool>,
```

Comment:
this one is camelCase at least in bitcoin core 29.0: https://bitcoincore.org/en/doc/29.0.0/rpc/wallet/walletcreatefundedpsbt/

```suggestion
    #[serde(rename = "lockUnspents", skip_serializing_if = "Option::is_none")]
    pub lock_unspents: Option<bool>,
```

Also note that we have plural, so it's `lockUnspents` (check the `s`)